### ISSUE TITLE
Crash after [🟥 Stop]

### ISSUE THREAD

**Issue by 0x13479** [vinegarhq/vinegar#445]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

No matter which renderer I use, Studio crashes few seconds after I click [🟥 Stop]. Only after that. If I don't run no simulations, I don't get crashes. v1.7.5 did not have this issue.

The log ends with:
```py
Oct  5 17:08:02.084 WRN Recieved signal signal=interrupt
Oct  5 17:08:02.084 WRN Killing Roblox pid=20
```
[Studio-2024-10-05T16:55:38Z.log](https://github.com/user-attachments/files/17266928/Studio-2024-10-05T16.55.38Z.log)

https://github.com/user-attachments/assets/b2b6c860-5a0a-4db0-9617-5b767250392a

### System information

```shell
* Vinegar: v1.7.6
* Distro: Freedesktop SDK 23.08 (Flatpak runtime)
* Processor: AMD Ryzen 5 5600X 6-Core Processor
  * Supports AVX: true
  * Supports split lock detection: false
* Kernel: 6.10.12-hardened1-1-hardened
* Wine (Player): wine-9.18 (Staging)
* Wine (Studio): wine-9.18 (Staging)
* Flatpak: [x]
* Cards:
  * Card 0: nvidia 0000:0a:00.0 /sys/class/drm/card1
```

**Comment by apprehensions** [comment]
It seems that Vinegar was sent a SIGTERM signal by another process, and this could be Roblox or your kernel. Please check your dmesg.

**Comment by 0x13479** [comment]
dmesg had no new lines. With strace I figured out that wineserver receives 3 SIGPIPEs at the moment it crashes (in the picture below).
![1728157587](https://github.com/user-attachments/assets/784f5a2c-1cd7-48ad-8869-bc3a7c4827cd)
The PID 23 is, from what I read, a kernel process, and I tried to disable the thing that kills processes which consume too much memory by:
```
echo "2" > /proc/sys/vm/overcommit_memory
```
But with that, Roblox crashes when entering the place with smth like "out of memory" message in the terminal.
I don't see this to be of any help so what I can do, is install a linux on VM with a normal kernel and then with linux-hardened to see whether that is the issue. i will do that later
